kilkari/src/test/java/org/motechproject/nms/kilkari/osgi/SubscriptionServiceBundleIT.java

```diff
@@ -66,12 +68,34 @@
     @Inject
     private InboxCallDataDataService inboxCallDataDataService;
 
+    private Language gLanguage;
+    private SubscriptionPack gPack1;
+    private SubscriptionPack gPack2;
+
+
+    private void setupData() {
+        cleanupData();
+        createLanguageAndSubscriptionPacks();
+
+        Subscriber subscriber1 = subscriberDataService.create(new Subscriber(1000000000L));
+        Subscriber subscriber2 = subscriberDataService.create(new Subscriber(2000000000L));
+
+        subscriptionService.createSubscription(subscriber1.getCallingNumber(), gLanguage, gPack1,
+                SubscriptionMode.IVR);
+        subscriptionService.createSubscription(subscriber2.getCallingNumber(), gLanguage, gPack1,
+                SubscriptionMode.IVR);
+        subscriptionService.createSubscription(subscriber2.getCallingNumber(), gLanguage, gPack2,
+                SubscriptionMode.IVR);
+    }
+
+
     private void createLanguageAndSubscriptionPacks() {
-        languageDataService.create(new Language("tamil", "10"));
+        gLanguage = languageDataService.create(new Language("tamil", "10"));
```

Comment:
I changed all of this from language to languageLocation.  It will be a race to see who submits first.